tendermint/src/lite_impl/signed_header.rs

```diff
@@ -48,6 +44,11 @@ impl lite::Commit for block::signed_header::SignedHeader {
     }
 
     fn validate(&self, vals: &Self::ValidatorSet) -> Result<(), Error> {
+        // TODO: self.commit.block_id cannot be zero in the same way as in go
```

Comment:
oh, in which case do we find `self.commit.block_id` to be zero in Go?

Reply:
I think I meant, we validate if it is zero the same way as in golang 
https://github.com/tendermint/tendermint/blob/ef56e6661121a7f8d054868689707cd817f27b24/types/block.go#L689-L691
because in the Struct here it is not an `Option`.